src/containerapp/azext_containerapp/_params.py

```diff
@@ -414,3 +414,19 @@ def load_arguments(self, _):
         c.argument('workload_profile_type', help="The type of workload profile to add or update. Run 'az containerapp env workload-profile list-supported -l <region>' to check the options for your region.")
         c.argument('min_nodes', help="The minimum node count for the workload profile")
         c.argument('max_nodes', help="The maximum node count for the workload profile")
+
+    # Patch
+    with self.argument_context('containerapp patch list') as c:
+        c.argument('resource_group_name', arg_type=resource_group_name_type)
+        c.argument('managed_env', options_list=['--environment', '-e'], help='Name or resource id of the Container App environment.')
+        c.argument('show_all', options_list=['--show-all'], help='Show all patchable and unpatchable container apps')
```

Comment:
It's no need to explicitly define the duplicate default option name
```suggestion
        c.argument('show_all', help='Show all patchable and unpatchable container apps')
```

Reply:
Updated -- Removed options_list for show-all option.